client/src/components/EditProfile/EditProfile.tsx

```diff
@@ -0,0 +1,214 @@
+                    fullWidth
+                    margin="normal"
+                    name="lastName"
+                    autoComplete="lastName"
+                    autoFocus
+                    variant="outlined"
+                    placeholder="Doe"
+                    helperText={touched.lastName ? errors.lastName : ''}
+                    error={touched.lastName && Boolean(errors.lastName)}
+                    value={values.lastName}
+                    onChange={handleChange}
+                  />
+                </Grid>
+                <Grid item md={3} xs={12} className={classes.inputLabelGridContainer}>
+                  <InputLabel htmlFor="email">Email Address</InputLabel>
+                </Grid>
+                <Grid item md={7} xs={12}>
+                  <TextField
+                    type="email"
+                    id="email"
+                    fullWidth
+                    margin="normal"
+                    name="email"
+                    autoComplete="email"
+                    autoFocus
+                    variant="outlined"
+                    placeholder="johndoe@gmail.com"
+                    helperText={touched.email ? errors.email : ''}
+                    error={touched.email && Boolean(errors.email)}
+                    value={values.email}
+                    onChange={handleChange}
+                  />
+                </Grid>
+                <Grid item md={3} xs={12} className={classes.inputLabelGridContainer}>
+                  <InputLabel htmlFor="phone">Phone Number</InputLabel>
+                </Grid>
+                <Grid item md={7} xs={12}>
+                  {isPhoneNumberAdded ? (
+                    <TextField
+                      type="tel"
+                      id="phone"
+                      fullWidth
+                      margin="normal"
+                      name="phone"
+                      autoComplete="phone"
+                      autoFocus
+                      variant="outlined"
+                      placeholder="Phone Number 647-777-702"
+                      helperText={touched.phone ? errors.phone : ''}
+                      error={touched.phone && Boolean(errors.phone)}
+                      value={values.phone}
+                      onChange={handleChange}
+                    />
+                  ) : (
+                    <Box
+                      marginTop={2}
+                      marginBottom={1}
+                      display="flex"
+                      justifyContent="space-between"
+                      alignItems="center"
+                    >
+                      <span className={classes.phoneSpan}>No phone number is added</span>
+                      <input type="text" hidden id="phone" />
+                      <Button
+                        type="button"
+                        variant="outlined"
+                        color="primary"
+                        onClick={() => setIsPhoneNumberAdded(true)}

```

Comment:
I would interpret an "Add a  Phone Number" button to mean the user could add multiple phone numbers. So you could change the state for this one to be an array of phone numbers, or you could remove the button and have this be a normal input.

Reply:
Done